subsystems/motor/speedmodifiers/AccelerationCap.java

```diff
@@ -41,15 +58,73 @@ public AccelerationCap(PDP pdp, double softStopVoltage, double hardStopVoltage)
 	 * This is designed to reduce power consumption (via voltage ramping)
 	 * and prevent RoboRIO/router brownouts.
 	 *
-	 * Default soft stop voltage is 11 volts.
-	 * Default hard stop voltage is 10 volts.
+	 * Default hard stop voltage is 7.0 volts.
 	 *
 	 * @param pdp
 	 *        The robot's power distribution panel.
 	 *        This is used to monitor the battery voltage.
 	 */
-	public AccelerationCap(PDP pdp) {
-		this(pdp, 11.0, 10.0);
+	public AccelerationCap(PDP pdp, int... channels) {
+		this(pdp, AccelerationCap.DEFAULT_HARD_STOP_VOLTAGE, channels);
+	}
+
+	protected double calculate(double inputSpeed) {
+		double deltaTime = (System.currentTimeMillis() - lastUpdate) / 1000.0;
+		lastUpdate = System.currentTimeMillis();
+		// Update current data
+		double newVoltageDrop = 0.0;
+		double lastVoltageDrop = 0.0;
```

Comment:
Why are there three variables? (newVoltageDrop, voltageDrop, and lastVoltageDrop)